### PR TITLE
Image added to the post

### LINKED ISSUE #720: Collection of bug reports/input from Mapping Fellows (Community Atlas)
Our Mapping Fellows (https://publiclab.org/tag/community-atlas) had a call today and we're reporting in a collection of UI/UX input from them here -- very helpful! Thanks Sairam and Mo!

I've separated these out into issues for Leaflet.DistortableImage and MapKnitter. The matching Leaflet.DistortableImage issue is at https://github.com/publiclab/Leaflet.DistortableImage/issues/306

Also cc #312 and #327!

### MapKnitter issues

* [x] replace the chat tab with a link to https://publiclab.org/chat
* [ ] include first image of a map in the "Share on PL" button in the post body field
* [x] remove "Legacy Editor" link and code
* [ ] Exporting - what is the purpose? Distinguish between "exporting", "sharing", "embedding", Sairam wasn't sure if "exporting" was for some internal purpose
* [ ] we should better distinguish (at least with tooltips) between "deleting" an image and "unplacing it" (where the latter is just taking it off the map, but leaving it in the sidebar. This may be a [MapKnitter issue](https://github.com/publiclab/mapknitter) in fact, not LDI) - #114 

Also aggregating open issues here that will address the updates requested by Mo in https://publiclab.org/notes/molangmuir10/06-10-2019/mapknitter-ui-evaluation?_=1560394957

* [ ] Reimplement a client side undo feature #72 (could also be an LDI issue)

### PR DESCRIPTION
**Pull request by divyabaid16**
Fixes #720 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

[//]: # (To mark checkbox write 'x' within the square brackets)

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/mapknitter-reviewers` for help, in a comment below

Thanks!

### CODE REVIEW

**Comment by divyabaid16** [comment]
The post after image is uploaded will look like:
Selection_188

File: app/views/layouts/knitter2.html.erb [hunk 1]
```diff
@@ -82,7 +82,7 @@
                 </button>
 
               <!-- Write a research note on Public Lab -->
-                <a target="_blank" class="btn btn-outline-secondary" title="Post a research note on Public Lab" href="//publiclab.org/post?title=MapKnitter map of <%= @map.name %>&tags=lat:<%= @map.lat %>,lon:<%= @map.lon %>,mapknitter&body=%3Ciframe%20src=%22https://mapknitter.org/embed/<%= @map.slug %>%22%20style=%22border:none%22%20width=%22100%25%22%20height=%22400px%22%3E%3C/iframe%3E%3Cbr /%3E%3Cbr /%3E<%= @map.description %>%3Cbr /%3Ehttps://mapknitter.org/embed/<%= @map.slug %>" type="button">
+                <a target="_blank" class="btn btn-outline-secondary" title="Post a research note on Public Lab" href="//publiclab.org/post?title=MapKnitter map of <%= @map.name %>&tags=lat:<%= @map.lat %>,lon:<%= @map.lon %>,mapknitter&body=<img src='<%= @map.warpables.first.image.url(:medium) %>'></img>%3Ciframe%20src=%22https://mapknitter.org/embed/<%= @map.slug %>%22%20style=%22border:none%22%20width=%22100%25%22%20height=%22400px%22%3E%3C/iframe%3E%3Cbr /%3E%3Cbr /%3E<%= @map.description %>%3Cbr /%3Ehttps://mapknitter.org/embed/<%= @map.slug %>" type="button">
```
Comment (jywarren) [inline app/views/layouts/knitter2.html.erb]: Here, do we have to encode the `<` characters as `%3C` and such? I saw you tested it, so maybe it just works, but I guess I'd worry about it rendering properly with nested `<` chars... did you just try the URL or did you click on it in the page? Thanks!!!

**Comment by divyabaid16** [comment]
Hi @jywarren 
I changed my code as per your suggestion and it's working fine.
Thank you for always guiding me :)
It's weird that it was working in both the ways with ```<``` and with ```%3C```.

**Comment by jywarren** [comment]
Yeah I dunno why! But ok sounds great. Thanks!!!

**Pull request merged**